### PR TITLE
Leaderboard: Hit api

### PR DESCRIPTION
**Pull request by Olivier-Kango**
#### In this particular project, I :
- Create a new game with the name of my choice by using the API.
    - Make sure that I saved the ID of my game that will be returned by API.
- Implement the "Refresh" button that will get all scores for a game created by me from the API (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button that will save a score for a game created by me (sending data to the API).
- Use `arrow functions` instead of the `function` keyword.
- Use `async` and `await` JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/setup.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
   recentSores.forEach((elt, index) => {
-    const { name, score } = elt;
+    const { user, score } = elt;
     const listScores = document.querySelector('.list-scores');
     const li = document.createElement('li');
-    li.innerText = `${name}: ${score}`;
+    li.innerText = `${user}: ${score}`;
     li.id = index + 1;
     listScores.appendChild(li);
   });
```
Comment (julie-ify) [inline src/modules/setup.js]: - [ ] Kindly change this method to an arrow function

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 /* eslint-disable no-unused-vars */
 import _, { add } from 'lodash';
 import './style.css';
-import recentSores from './modules/setup.js';
+import {
+  gameScore, button, addScore, submitButton,
+} from './modules/hit-api.js';
 /* eslint-enable no-unused-vars */
```
Comment (julie-ify) [inline src/index.js]: - [ ] Kindly remove the linters disabling here. Linters help us write standardized and maintain code

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Olivier-Kango,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hello @Olivier-Kango 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**